app/controllers/conferences_controller.rb

```diff
@@ -61,11 +61,7 @@ def show
         @booths = @conference.confirmed_booths.order('title')
       end
       if splashpage.include_happening_now
```

Comment:
minor but it's nice if boolean methods end in `?`, they should be automatically defined by rails. :)